README.md

```diff
@@ -39,8 +39,8 @@ I will use a relational database to create the data structure for a vet clinic.
 ### Key Features <a name="key-features"></a>
 
 - **Follow a correct Gitflow**
-- **Use database transactions**
-- **Modify and delete data in SQL**
+- **Use primary key & foreign key mechanism for joining tables**
+- **Query multiple tables**
```

Comment:
![image](https://github.com/CarlosIgreda/Vet-clinic-database/assets/78251927/519c9ec3-49cc-4d05-b210-35a9792ddb98)

![image](https://github.com/CarlosIgreda/Vet-clinic-database/assets/78251927/3d57e6f4-1b86-4ab8-8303-1205928916da)

Kindly fix the broken license link, please let's avoid broken links in our readme for a professional repository. 🤗🛠

Reply:
@abdulhamiid Done it.